### ISSUE TITLE
[CLI] Where can I find an example of ~/.cdk.json file configuration

### ISSUE THREAD

**Issue by robertd** [aws/aws-cdk#10430]
### The Question

Hello,

I'm trying to use newly issued [fix for progress bar vs events](https://github.com/aws/aws-cdk/pull/9623) on a more global level. I've tried creating `~/.cdk.json` but I can't seem to get it right. I haven't seen any examples in docs showing how to properly configure `~/.cdk.json` globally. 

Can you please provide an example how to do this?

Thanks!

### Environment

  - **CDK CLI Version:** 1.63.0
  - **Node.js Version:** v14.11.0
  - **OS:** OSX Mojave
  - **Language (Version):** TypeScript (3.8.3)

**Comment by NetaNir** [comment]
You should have a `cdk.json` file in your directory, it is created by the `cdk init` when creating a cdk application, In any case you can execute `cdk init` in an empty folder to get the file.

**Comment by robertd** [comment]
Hello, @NetaNir.  I'm not referring to a new project here. I was referring to a global environment (`~/.cdk.json`) as stated in the documentation (5th bullet). 

https://docs.aws.amazon.com/cdk/latest/guide/context.html

![image](https://user-images.githubusercontent.com/31543/93658009-d7700280-f9f4-11ea-8531-be72d19ff8d3.png)

I'd really like to declare variables on a global level so that all of my projects are using `--progress=events` setting introduced in https://github.com/aws/aws-cdk/pull/9623.

**Comment by robertd** [comment]
@NetaNir Here are the contents of my `~/.cdk.json` file, but they're being ignored when I run `cdk init app` in a new project. What am I doing wrong?

```json
{
  "context": {
    "language": "typescript",
    "progress": "events"
  }
}
```

**Comment by NetaNir** [comment]
@robertd Thanks for debugging this. I think that this is the expected behavior, the local `cdk.json` take precedence over the global one. Merging logic will most likely be more confusing. We definitely need to document the order in which the context values are populated to out docs. @Jerry-AWS

**Comment by kpeters-cbsi** [comment]
Merge logic may be confusing, but without it, `~/.cdk.json` is pretty well useless. As a developer who works on a number of different small projects in CDK, I would expect `~/.cdk.json` to contain global configuration values - such as setting progress to bar vs. events, or a default language, and the local `cdk.json` to contain local configuration and overrides. Otherwise, as I said above, what's the point of `~/.cdk.json`?

**Comment by bitbauer** [comment]
@kpeters-cbsi @robertd Your comments on expected behavior of merging context from ~/.cdk.json are relating to #10823
(... where I have added a PR)

**Comment by robertd** [comment]
@bitbauer Sweet. Thank you. 🙏

**Comment by robertd** [comment]
Closed by #10823
